cpp/src/neighbors/detail/knn_brute_force.cuh

```diff
@@ -23,16 +24,25 @@
 #include "./fused_l2_knn.cuh"
 #include "./haversine_distance.cuh"
 #include "./knn_merge_parts.cuh"
+#include "./knn_utils.cuh"
 
```

Comment:
```suggestion

#include <raft/core/bitmap.cuh>
```
Include the bitmap CUDA header file where the CUDA functions are called.

Reply:
Thank you @lowener, very helpful, I fixed it!